packages/client/src/api/wargames_api/index.ts

```diff
@@ -157,22 +157,26 @@ export const listenForWargameChanges = (name: string, dispatch: PlayerUiDispatch
 }
 
 export const pingServer = (): Promise<any> => {
-  return fetch(serverPath + 'healthcheck')
-  .then((response: Response): Promise<any> => response.json())
-  .then((data: any) => {
-    return data.status
-  })
-  .catch((err) => {
-    console.log(err)
-    return "NOT_OK"
-  })
-} 
+  return fetch(serverPath + 'health_check')
+    .then((response: Response): Promise<any> => response.json())
+    .then((data: any) => {
+      // check the version returned from the server with the current client version
+      if (!data.version || data.version !== process.env.REACT_APP_VERSION) {
+        return "NOT_OK"
```

Comment:
@lebaphi  - we're probably going to have to check for `"NOT_OK"` in the client code too.  So, I think we should create a const somewhere for it - to ensure a type doesn't break the app.

Reply:
sure, I will update now @IanMayo